src/plugins/intel_gna/common/versioning.cpp

```diff
@@ -0,0 +1,21 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "versioning.hpp"
+
+#include <sstream>
+#include <string>
+
+#include "openvino/core/version.hpp"
+
+namespace GNAPluginNS {
```

Comment:
Let's use `ov::intel_gna` namespace for new code

Reply:
OK